web/src/OverviewTable.tsx

```diff
@@ -779,13 +851,22 @@ export function TableWithoutGroups({ resources, buttons }: TableWrapperProps) {
   }, [resources, buttons])
   const columns = getTableColumns(features)
 
+  let totalOrder = enabledRowsFirst(data)
+  let firstResourceName = totalOrder[0]?.name
+  let [focused, setFocused] = useState(firstResourceName || "")
```

Comment:
i do find it a little odd that the first row is selected before i (as a user) have tried to navigate to any resources. wdyt about only selecting the first resource once a user tries to navigate via shortcuts?

i could potentially see users getting confused about the visual signal if they don't know about keyboard nav or they don't find it later on.

Reply:
Makes sense !

Reply:
i was trying to replicate the gmail behavior here, where, if you have keyboard shortcuts enabled, it focuses the first row on pageload.

are you saying that focus state should only appear once you start using the keyboard? that seems like it would make it hard to discover...